lib/stemcell/option_parser.rb

```diff
@@ -187,6 +187,12 @@ class OptionParser
         :type  => String,
         :env   => 'CHEF_DATA_BAG_SECRET_PATH'
       },
+      {
+        :name => 'cpu_options',
+        :desc => "cpu options",
```

Comment:
nit: can you make this help text more descriptive?
```suggestion
        :desc => "comma-separated list of cpu option key=value pairs",
```